evaluator/evaluate.go

```diff
@@ -258,6 +273,8 @@ func updateSystemPlatform(tx *gorm.DB, system *models.SystemPlatform, old, new S
 	data["advisory_bug_count_cache"] = counts[2]
 	data["advisory_sec_count_cache"] = counts[3]
 	data["last_evaluation"] = time.Now()
+	data["packages_installed"] = installed
+	data["packages_updatable"] = updatable
```

Comment:
I know there were some issues when updating "default values", e.g. 0 for int. Are you sure it will work here?

Reply:
It should. We are already using it for advisory counts.